gtsfm/utils/ellipsoid.py

```diff
@@ -0,0 +1,108 @@
+"""Algorithms to center and align 3D points and camera frustums to the x, y, and z axes using SVD. Used in React
+Three Fiber Visulization Tool.
+
+Authors: Adi Singh
+"""
+
+import numpy as np
+from typing import Tuple
+from gtsam import Pose3, Rot3
+
+from gtsfm.common.gtsfm_data import GtsfmData
+
+
+def transform_point_cloud_wrapper(gtsfm_data: GtsfmData) -> Pose3:
```

Comment:
thanks for all the updates Adi, looks quite good. Almost ready for merge I think.

Let's try to think of a more descriptive name for this function, since `wrapper` is a bit vague.

Reply:
Ok, I'm changing it to `get_point_cloud_alignment_transform` since the function is not even transforming the point cloud anymore. It's just calculating and returning `walignedTw`

Reply:
how about `get_ortho_axis_alignment_transform()` ? (since we're aligning both camera poses and points)

Reply:
Sounds good to me